tutorials/hpc/01_basics/01_basics_dndarrays.py

```diff
@@ -0,0 +1,36 @@
+import heat as ht
+
+# ### DNDarrays
+#
+#
+# Similar to a NumPy `ndarray`, a Heat `dndarray`  (we'll get to the `d` later) is a grid of values of a single (one particular) type. The number of dimensions is the number of axes of the array, while the shape of an array is a tuple of integers giving the number of elements of the array along each dimension.
+#
+# Heat emulates NumPy's API as closely as possible, allowing for the use of well-known **array creation functions**.
+
+
+a = ht.array([1, 2, 3])
+print("array creation with values [1,2,3] with the heat array method:")
+print(a)
+
+a = ht.ones(
+    (
+        4,
+        5,
+    )
+)
```

Comment:
```suggestion
# fmt: off
a = ht.ones((4, 5,))
# fmt: on
```

This is a way I've found to disable black on specific lines. But it might be best to disable black on this entire directory.

Reply:
@mrfh92  I just noticed you already excluded the tutorials directory from the linting on this branch. It works locally but we need to merge the change into main first, see #1748 , otherwise pre-commit on remote will still reformat.